services/post.service.js

```diff
@@ -1,23 +1,16 @@
+import { Post } from '../db/models/post.js';
 import { EntityService } from './entity.service.js';
 
-export class PostService {
-  constructor(db) {
-    this.service = new EntityService(db, 'post');
+export class PostService extends EntityService {
+  constructor() {
+    super(Post);
   }
 
-  async createPost(content, authorId) {
-    return this.service.create({ content, authorId });
-  }
-
-  async getPost(id) {
-    return this.service.getOne(id);
-  }
-
-  async deletePost(id) {
-    return this.service.deleteById(id);
-  }
+  async getPostsByUserId(authorId) {
```

Comment:
You can use inherited search/getMany/find(described below) method instead of getPostsByUserId.